PaddleCV/object_detection/docs/MODEL_ZOO.md

```diff
@@ -9,39 +9,41 @@
 
 ## Common settings
 
-- All models below except SSD were trained on `coco_2017_train`, and tested on the `coco_2017_val`.
-- The Batch Normalization layer of the backbone is replaced by the Affine Channel layer.
-- Unless otherwise noted, all ResNet are adopted [ResNet-B](https://arxiv.org/pdf/1812.01187) as the backbone.
+- All models below except SSD were trained on `coco_2017_train`, and tested on `coco_2017_val`.
+- Batch Normalization layers in backbones are replaced by Affine Channel layers.
+- Unless otherwise noted, all ResNet backbones adopt the [ResNet-B](https://arxiv.org/pdf/1812.01187) variant..
 - For RCNN and RetinaNet models, only horizontal flipping data augmentation was used in the training phase and no augmentations were used in the testing phase.
 
 ## Training Schedules
 
-- We adopt the exactly same training schedules as [Detectron](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md#training-schedules). 
-- 1x indicates the schedule starts at a LR of 0.02 and is decreased by a factor of * 0.1 after 60k and 80k iterations and finally terminates at 90k iterations for minibatch size 16. 
-- 2x indicates twice as long as the 1x schedule with the LR change points scaled proportionally.
+- We adopt exactly the same training schedules as [Detectron](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md#training-schedules). 
+- 1x indicates the schedule starts at a LR of 0.02 and is decreased by a factor of 10 after 60k and 80k iterations and eventually terminates at 90k iterations for minibatch size 16 or starts at a LR of 0.01 and is decreased by a factor of 10 after 120k and 160k iterations and eventually terminates at 180k iterations for minibatch size 8.
```

Comment:
man...this is lazy...

Reply:
For batch size 8, LR is decreased to 0.01, total training iterations are doubled, and the decay milestones are scaled by 2.